.github/workflows/test-suite.yml

```diff
@@ -71,7 +73,7 @@ jobs:
     with:
       terraform_version: ${{ matrix.terraform_version }}
       provider_version: ${{ matrix.provider_version }}
-      atlas_cloud_env: ${{ inputs.atlas_cloud_env }}
+      atlas_cloud_env: ${{ inputs.atlas_cloud_env || github.event.schedule == '0 0 2-31 * 0' && 'qa' || 'dev' }}
```

Comment:
When is https://jira.mongodb.org/browse/CLOUDP-268251 planned to be addressed?

Reply:
I just started on it. Hopefully, I'll complete it tomorrow